diff --git a/dist/compiler/runtime/localize-url.js b/dist/compiler/runtime/localize-url.js
index da74082ec1da8de3497d1a09fc7f36e54c905d50..747ca6b6771cb9cfd705026202029e01d5ad020e 100644
--- a/dist/compiler/runtime/localize-url.js
+++ b/dist/compiler/runtime/localize-url.js
@@ -178,7 +178,7 @@ export function deLocalizeUrl(url) {
     return urlObj;
 }
 /**
- * De-localizes a URL using the default pattern (/:locale/*)
+ * De-localizes a URL using the default pattern (/:locale/...)
  * @param {string|URL} url
  * @returns {URL}
  */
